lib/bolt/node/ssh.rb

```diff
@@ -14,6 +14,16 @@ def self.initialize_transport(logger)
       }
     end
 
+    if Gem.win_platform?
```

Comment:
Don't use `Gem.win_platform?` - instead use `!!File::ALT_SEPARATOR` to check for Windows like we do in https://github.com/puppetlabs/puppet/blob/ebd96213cab43bb2a8071b7ac0206c3ed0be8e58/lib/puppet/util/platform.rb#L10

This is compiled into Ruby itself and won't change.

Reply:
Will do!

Though I found the same in bolt and pdk, and most importantly in:

https://github.com/net-ssh/net-ssh/blob/master/lib/net/ssh/authentication/agent.rb#L9

which I modeled.